CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ Changes since v2.17
 ### Additions
 - REMS now supports PostgreSQL version 13. (#2642)
 - Experimental GA4GH Permissions API now allows users to query their own permissions via `/api/permissions/:user`. (#2631)
+- The handler can now change the applicant of a submitted application. (#2581)
+  - This feature can be disabled by adding `:application.command/promote-to-applicant` to `:disable-commands`.
```

Comment:
Hmm which should be the default?

Reply:
config-defaults.edn doesn't disable any commands currently, so I thought I'd follow that pattern. One alternative would be to have an explicit feature flag for this feature if we consider it experimental.

Reply:
We should let @jaakkocsc decide the default (is it ok to add as default new command, and perhaps disable by hand in the environments CSC manages)

Reply:
Discussed this on chat, agreed with Jaakko that enabling by default is ok. We'll just point people at the changelog and let them test the new feature in their demo environment before deciding whether to disable it or not. Enabling by default is simpler codewise.